module/chunks/chunkVerifier_test.go

```diff
@@ -48,8 +48,8 @@ var eventsList = flow.EventsList{
 
 // the chain we use for this test suite
 var testChain = flow.Emulator
-var epochSetupEvent, _ = convert.EpochSetupFixture(testChain)
-var epochCommitEvent, _ = convert.EpochCommitFixture(testChain)
```

Comment:
This is causing functions in the unittest to be used into application package.

I fixed it by moving fixtures to a separate `fixtures` package.